### PR TITLE
Add rel=canonical to templates

### PR DESCRIPTION
**Pull request by phamann**
Adds two bits of meta data to page templates:

- `<link rel=canonical>` for robots to know about genuine content pages
- `<meta name=cache-key>` for the client-side JS to post message to the ServiceWorker to ascertain latest cache time.

### CODE REVIEW

File: server/controllers/content.js [hunk 1]
```diff
@@ -18,7 +18,9 @@ router.get('/:uuid', (req, res, next) => {
 
 	const options = setNavigation({
 		layout: 'chocolate',
-		withAssetPrecache: true
+		withAssetPrecache: true,
+		cacheKey: `/content/${uuid}`,
```
Comment (leggsimon) [inline server/controllers/content.js]: should have `https://www.ft.com`

File: server/controllers/no-preflight.js [hunk 2]
```diff
@@ -6,6 +6,7 @@ const landingRouter = new Router();
 landingRouter.get('/landing', (req, res) => {
 	res.render('landing', {
 		withAssetPrecache: true,
+		cacheKey: '/__offline/landing',
```
Comment (leggsimon) [inline server/controllers/no-preflight.js]: should have `https://www.ft.com`

File: server/controllers/top-stories.js [hunk 3]
```diff
@@ -11,7 +11,8 @@ module.exports = (req, res) => {
 		},
 		layout: 'chocolate',
 		withAssetPrecache: true,
-		noRelatedContent: true
+		noRelatedContent: true,
+		cacheKey: '/__offline/top-stories'
```
Comment (leggsimon) [inline server/controllers/top-stories.js]: should have `https://www.ft.com`

File: server/controllers/no-preflight.js [hunk 4]
```diff
@@ -6,7 +6,8 @@ const landingRouter = new Router();
 landingRouter.get('/landing', (req, res) => {
 	res.render('landing', {
 		withAssetPrecache: true,
-		cacheKey: '/__offline/landing'
+		cacheKey: '/__offline/landing',
+		layout: 'chocolate'
```
Comment (quarterto) [inline server/controllers/no-preflight.js]: That was fixed by #29, is this branch out of date?

**Pull request merged**